dnf-behave-tests/features/repoquery/main.feature

```diff
@@ -354,6 +354,40 @@ Scenario: repoquery --extras NAME (package is not installed)
  Then the exit code is 0
   And stdout is empty
 
+   # --extras: installed pkgs, different NEVRA in available repository
+Scenario: dnf repoquery --extras (when there are such pkgs with different NEVRA in repository)
+ Given I drop repository "repoquery-main"
+ When I execute rpm with args "-i --nodeps {context.dnf.fixturesdir}/repos/repoquery-multilib/x86_64/mid-a1-1.0-1.x86_64.rpm"
+  Then the exit code is 0
+ When I execute dnf with args "repoquery --installed"
+  Then stdout is
```

Comment:
The indentation is mixed in these steps (in all scenarios you've added), can you fix it?